java/pgv-java-stub/src/test/java/io/envoyproxy/pgv/StringValidationTest.java

```diff
@@ -212,20 +218,33 @@ public void uriRefWorks() throws ValidationException {
 
     @Test
     public void uuidWorks() throws ValidationException {
+        char[] chars = "0123456789abcdefABCDEF".toCharArray();
```

Comment:
A comment explaining that you are testing UUIDs of number, lower, and upper character would be helpful.